src/Sarif/Schemata/sarif-schema.json

```diff
@@ -367,6 +367,30 @@
       "required": [ "threadFlows" ]
     },
 
+    "configurationOverride": {
+      "description": "Information about how a specific tool report was reconfigured at runtime.",
```

Comment:
>tool report [](start = 55, length = 11)

If you want to use the spec's terminology, "tool report" => "reporting item". But rather than using that obscure coinage, you could just say "tool report" => "rule or notification".